scripts/testnet/include/config.sh

```diff
@@ -20,7 +20,8 @@ generateConfig() {
     -num-of-observers-in-metachain $TMP_META_OBSERVERCOUNT    \
     -metachain-consensus-group-size $META_CONSENSUS_SIZE      \
     -stake-type $GENESIS_STAKE_TYPE \
-    -hysteresis $HYSTERESIS
+    -hysteresis $HYSTERESIS \
+    -round-duration $ROUND_DURATION_IN_MS
```

Comment:
from mx-chain-deploy-go:
```
roundDuration = cli.UintFlag{
		Name:  "round-duration",
		Usage: "round duration in miliseconds",
		Value: 5000,
	}
```
Previously, if no flag value was present, the round duration would have been 5 seconds, instead of 6 after your changes. Either set ROUND_DURATION_IN_MS to 5000, or update mx-chain-deploy-go's default value

Reply:
I know, but the node's original (and defined) value is 6000 (mainnet like)
I would want to keep it set to 6000. Whoever wants to alter it, can use the `local.sh` file and put whatever value he/she want